tests/Expected/Business/testJsonSerialize.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
I can rename the folder, but i think it should be called `Expected` since it contains the expected output from `json_encode()`
Fixtures would be for preparing the test data.